src/farcasterd/runtime.rs

```diff
@@ -943,12 +455,12 @@ impl Runtime {
                     }
                     OfferStatusSelector::InProgress => {
                         let pub_offers = self
```

Comment:
incidentally, also more efficient for a long-running node with large histoy of swaps